inst/css/BS4/pkgdown.sass

```diff
@@ -198,6 +155,14 @@ h6.dropdown-header {
 
 /* Clipboard --------------------------*/
 
+.btn-primary:hover,
```

Comment:
Isn't this more generic than just the clipboard? Why do we have to override so many colours?

Reply:
* I've now made it more specific.
* By default the hover background color is a simply a bit darker than the default color, @apreshill suggested we made this depend on secondary which makes the difference between default and hover more striking.